packages/perseus/src/widgets/numeric-input.tsx

```diff
@@ -215,8 +213,9 @@ export class NumericInput extends React.Component<Props, State> {
 
         if (!result) {
             // Otherwise, if the guess is not correct
-            const otherAnswers = [].concat(
-                // @ts-expect-error [FEI-5003] - TS2769 - No overload matches this call.
+            const otherAnswers = (
+                [] as ReadonlyArray<PerseusNumericInputAnswer>
+            ).concat(
                 rubric.answers.filter((answer) => answer.status === "ungraded"),
                 rubric.answers.filter((answer) => answer.status === "wrong"),
             );
```

Comment:
We could use spread here instead of `[].concat()` if we wanted to.

Reply:
That seems more friendly as we'd avoid having to do the casting of the initial array, right?